src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        //Create user variable
```

Comment:
redundant comment because this is a fairly self-explanatory code that doesn't need further explanation